dev/how-to/maintain/identify-slow-queries.md

```diff
@@ -35,21 +30,38 @@ select count(1) from t_slim, t_wide where t_slim.c0>t_wide.c0 and t_slim.c1>t_wi
+* `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, while `false` indicates that the SQL statement is executed by the user.
+* `Index_ids`: The IDs of the indexes involved in the statement.
+* `Succ`: Whether the statement executes successfully.
+* `Backoff_time`: The waiting time before retry when this statement encounters errors that require a retry. The common errors as such include: `lock occurs`, `Region split`, and `tikv server is busy`.
+
+Memory usage fields:
+
+* `Memory_max`: The maximum memory space used during the execution period of this SQL statement (the unit is byte).
+
+User fields:
+
 * `User`: The name of the user who executes this statement.
 * `Conn_ID`: The Connection ID (session ID). For example, you can use the keyword `con:3` to `grep` the log whose session ID is 3.
 * `DB`: The current database.
-* `Index_ids`: The IDs of the indexes involved in the statement.
-* `Is_internal`: Whether the SQL statement is TiDB internal. `true` indicates that the SQL statement is executed internally in TiDB, such as `analyze`, `load variables`, etc. `false` indicates that the SQL statement is executed by the user.
-* `Digest`: The fingerprint of the SQL statement.
-* `Memory_max`: The maximum memory space used during the execution period of this SQL statement (the unit is byte).
-* `Query_time`: Indicates the execution time of this statement. Only the statement whose execution time exceeds slow-threshold outputs this log (the unit is second). The unit of all the following time fields is second.
-* `Process_time`: The total processing time of this SQL statement in TiKV. Because the data is sent to TiKV concurrently, this value may exceed `Query_time`.
-* `Wait_time`: The total waiting time of this statement in TiKV. Because the Coprocessor of TiKV runs a limited number of threads, requests might queue up when all threads of Coprocessor are working. When a request in the queue takes a long time to process, the waiting time of the subsequent requests will increase.
-* `Backoff_time`: The waiting time before retry when this statement encounters errors that require a retry. The common errors as such include: `lock occurs`, `Region split`, and `tikv server is busy`.
+
+TiKV Coprocessor Task fields:
+
 * `Request_count`: The number of Coprocessor requests that this statement sends.
 * `Total_keys`: The number of keys that Coprocessor has scanned.
+* `Process_time`: The total processing time of this SQL statement in TiKV. Because the data is sent to TiKV concurrently, this value may exceed `Query_time`.
+* `Wait_time`: The total waiting time of this statement in TiKV. Because the Coprocessor of TiKV runs a limited number of threads, requests might queue up when all threads of Coprocessor are working. When a request in the queue takes a long time to process, the waiting time of the subsequent requests will increase.
```

Comment:
```suggestion
* `Wait_time`: The total waiting time of a statement in TiKV. Because the Coprocessor of TiKV runs a limited number of threads, requests might queue up when all threads of Coprocessor are working. When a request in the queue takes a long time to process, the waiting time of the subsequent requests increases.
```